gallery/src/GalleryNav/GalleryNav.scss

```diff
@@ -0,0 +1,3 @@
+.gallery-nav-link {
+  color: var(--nx-color-text-dark);
```

Comment:
Would this not be applicable anywhere that an `NxCollapsibleItems` of links is used?

Reply:
I think that's the safest choice, and don't see any issue with it being applicable anywhere ` NxCollapsibleItems ` is used. I'll make a ticket for that and edit the Collapsible Items page in Kaleido.

Reply:
> I think that's the safest choice, and don't see any issue with it being applicable anywhere `NxCollapsibleItems` is used. I'll make a ticket for that and edit the Collapsible Items page in Kaleido.

https://issues.sonatype.org/browse/RSC-862